gatsby-site/.gitignore

```diff
@@ -2,6 +2,7 @@
 .cache
 node_modules
 yarn-error.log
+package-lock.json
```

Comment:
Contrary to what it may seem, it's a good idea to commit the package-lock.json file as it makes your dependency try more accurate across installs. If possible, I'd remove it from the gitignore.

Reply:
✓